run/storage-streaming/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "nodejs-docs-samples-http-storage-streaming",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
+  },
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "scripts": {
+    "test": "mocha test/*.test.js --timeout=60000 --exit"
+  },
+  "dependencies": {
+    "@google-cloud/storage": "^5.8.5",
+    "busboy": "^0.3.1",
+    "mocha": "^8.0.0",
+    "uuid": "^8.3.2"
+  },
+  "devDependencies": {
+    "child-process-promise": "^2.2.1",
```

Comment:
add functions framework dependency?

Reply:
Done - I added NPM `script` entries to run each sample with the FFs.